src/WebJobs.Script.WebHost/WebHostServiceCollectionExtensions.cs

```diff
@@ -66,6 +66,7 @@ public static void AddWebJobsScriptHost(this IServiceCollection services, IConfi
             services.AddStandbyServices();
 
             // Core script host services
+            services.TryAddSingleton<IScriptJobHostEnvironment, WebScriptJobHostEnvironment>();
```

Comment:
Had to move this from its previous location (see below). It turns out that we actually have a bug currently where the WebJobsScriptHostService environment is null on resolution.